web/src/routes/(user)/albums/[albumId=id]/[[photos=photos]]/[[assetId=id]]/+page.svelte

```diff
@@ -275,12 +276,15 @@
     return;
   };
 
-  const refreshAlbum = async () => {
+  const refreshAlbum = async (options: { assets?: AssetResponseDto[] } = {}) => {
```

Comment:
The optional parameter could be removed if `withoutAssets` is set to `false` (depending on how expensive the query is in the backend).